content/sensu-go/5.16/guides/email-handler.md

```diff
@@ -0,0 +1,204 @@
+
+{{< highlight shell >}}
+sensuctl asset list
+{{< /highlight >}}
+
+You should see the `sensu/sensu-email-handler` asset in the list.
+For a detailed list of everything related to the asset that Sensu added automatically, run:
+
+{{< highlight shell >}}
+sensuctl asset info sensu/sensu-email-handler
+{{< /highlight >}}
+
+## Add an event filter
+
+Event filters allow you to fine-tune how your events are handled and [reduce alert fatigue][7].
+In this guide, your event filter will send notifications only when an event's state changes (for example, from `warning` to `critical` or from `critical` to `ok`).
+
+Here's an overview of how the `state_change_only` filter will work:
+
+- If CPU usage reaches 75%, you will receive **one** email notification for the change to `warning` status
+- If CPU usage stays at 75% for the next hour, you **will not** receive repeated email notifications every 60 seconds during that hour
+- If CPU usage increases to 90% after 1 hour at 75%, you will receive **one** email notification for the change from `warning` status to `critical` status
+- If CPU usage fluctuates betwen 50% and 95% for the next hour, you **one** email notification each time the status changes
+
+Adding the event filter requires two parts:
+
+1. Create the event filter
+2. Create the email handler definition to specify the email address where the `sensu/sensu-email-handler` asset will send incident notifications
+
+First, to create the event filter, run:
+
+{{< highlight shell >}}
+cat << EOF | sensuctl create
+---
+type: EventFilter
+api_version: core/v2
+metadata:
+  annotations: null
+  labels: null
+  name: state_change_only
+  namespace: default
+spec:
+  action: allow
+  expressions:
+  - event.check.occurrences == 1
+  runtime_assets: []
+EOF
+{{< /highlight >}}
+
+Second, create the email handler definition.
+In the handler definition's `command` value, you'll need to change a few things.
+
+Copy this text into a text editor:
+
+{{< highlight shell >}}
+cat << EOF | sensuctl create
+---
+api_version: core/v2
+type: Handler
+metadata:
+  namespace: default
+  name: email
+spec:
+  type: pipe
+  command: sensu-email-handler -f YOUR-SENDER@example.com -t YOUR-RECIPIENT@example.com -s YOUR-SMTP-SERVER.example.com
+    -u USERNAME -p PASSWORD
+  timeout: 10
+  filters:
+  - is_incident
+  - not_silenced
+  - state_change_only
+EOF
+{{< /highlight >}}
+
+Then, replace the following text:
+- `YOUR-SENDER@example.com`: Replace with the email address you want to use to send email alerts
+- `YOUR-RECIPIENT@example.com`: Replace with the email address you want to receive email alerts
+- `YOUR-SMTP-SERVER.example.com`: Replace with the hostname of your SMTP server
+- `USERNAME`: Replace with your SMTP username, typically your email address
+- `PASSWORD`: Replace with your SMTP password, typically the same as your email password
+
+_**NOTE**: To use Gmail or G Suite as your SMTP server, follow Google's instructions for [sending email via SMTP][14]._
```

Comment:
```suggestion
_**NOTE**: To use Gmail or G Suite as your SMTP server, follow Google's instructions for [sending email via SMTP][14]. Also be aware that if you have 2fa enabled on your Google account, you'll need an [app password][15] instead of your login password._
```